tests/positive/issue1731/builtinTrace.juvix

```diff
@@ -8,7 +8,7 @@ axiom trace : {A : Type} → A → A;
 
 terminating
 f : Nat → Nat → Nat;
-f x y := if (x == 0) y (trace x >>> (f (sub x 1) y));
+f x y := if (x == 0) y (trace x >>> f (sub x 1) y);
```

Comment:
Just curiosity, did you change this manually? or it was the formatter?

Reply:
That is after `make pre-commit` command work, not sure how it was not caught earlier

Reply:
Exactly. pre-commit locally runs the Juvix formatter and the CI as well. I'm supprised, a new mystery to my stack.